components/blocks/index.tsx

```
@@ -0,0 +1,197 @@
+        <GroupContent style={style} children={children} path={dataPath} />
+      </CardContent>
+    </Card>
+  ),
+  checkbox: ({
+    id,
+    label,
+    dataPath,
+    labelStyle,
+    readOnly,
+  }: CheckboxBlockType) => {
+    const [value, setValue] = useSheetValue(dataPath!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)